sqld/src/lib.rs

```
@@ -82,7 +85,8 @@ where
     handles.push(tokio::spawn(server.serve(factory)));
 
     if let Some(addr) = config.http_addr {
-        let authorizer = http::auth::parse_auth(config.http_auth)?;
+        let authorizer =
+            http::auth::parse_auth(config.http_auth).map_err(Error::AuthConfigError)?;
```

the `parse_auth` method should return the appropriate error